.github/workflows/integration-test-iOS16.yaml

```diff
@@ -1,4 +1,4 @@
-name: "Integration Test: iOS 14.4"
+name: "Integration Test: iOS 16"
```

Comment:
Could we keep this named as iOS 16.0 for consistency with the other files (and the README)? And change the file name to `…iOS16_0.yaml`. It'll make it easier to bump the version number to 16.something in the future.

Reply:
Good point. Here goes another round of cancelling workflows game 😆

Reply:
https://github.com/ably/ably-cocoa/pull/1543/commits/3e62e0d3ef7a9ec95a479059a0a491f95fdbaf40